common/environment/environment.cpp

```diff
@@ -922,7 +922,12 @@ class CConstDropZoneInfo : public CConstEnvBase, implements IConstDropZoneInfo
 public:
     IMPLEMENT_IINTERFACE;
     IMPLEMENT_ICONSTENVBASE;
-    CConstDropZoneInfo(CLocalEnvironment *env, IPropertyTree *root) : CConstEnvBase(env, root) {}
+    CConstDropZoneInfo(CLocalEnvironment *env, IPropertyTree *root) : CConstEnvBase(env, root)
+    {
+        StringBuffer posixPath;
+        getStandardPosixPath(posixPath, root->queryProp("@directory"));
+        root->setProp("@directory", posixPath.str());
```

Comment:
You should not call setX on the trees.
If there tree is from Dali - you will be altering the contents there.

Reply:
Changed